spk/watcher3/src/dsm-control.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
+
+# Package
+PACKAGE="watcher3"
+DNAME="Watcher3"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python3"
+GIT_DIR="/usr/local/git"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${GIT_DIR}/bin:${PATH}"
+USER="watcher3"
+PYTHON="${INSTALL_DIR}/env/bin/python3"
+WATCHER3="${INSTALL_DIR}/share/Watcher3/watcher.py"
+CFG_FILE="${INSTALL_DIR}/var/config.cfg"
```

Comment:
that file doesn't exist does it? afaik the file is created on startup but most tools provide a simple configuration file which defines at least port, host and that the browser shouldn't be started see couchpotato for example: https://github.com/SynoCommunity/spksrc/blob/master/spk/couchpotatoserver/src/settings.conf

i guess you'll have to define the directory where to save the database which should be `${INSTALL_DIR}/var/` where all data of the tool should be stored to make sure that data presists during updates

Reply:
According to check manual of wizard3 this file gets created if it does not exist. I think it is good to specify so we know where the file reside.

Reply:
it sure is. but most tools have an option to specify where to save the database and some other stuff (mostly called data-dir or something like that) the way you are currently running the app will result in a deleted database when the SPK is updated!

So either use a "base config" which povides those options. or use the flags provided by watcher: ex: `--log ${INSTALL_DIR}/var/ --db ${INSTALL_DIR}/var/database.sqlite` personally i'd prefer to go with the default config as we do this for most SPK's already

Reply:
as you have already installed and running it. the simplest way would be to get the config of your setup, strip it down to the most basic usage (data directory, port, host, database etc) and put that in the `src` folder and copy it over on installation like cp and others do it ;)

Reply:
Ok, you win :). I've updated config.cfg with bare config now.